quick-start-guide/.idea/artifacts/saml2_web_app_dispatch_war.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ArtifactManager">
```

Comment:
These files are InteliJ idea specific project files. Remove them from the PR and configure the .gitignore file to exclude them (.idea directory and *.iml files) from any future commit. 

https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/master/.gitignore can be used as a reference